tests/unittests/JITTest.cpp

```diff
@@ -561,6 +561,78 @@ TEST_P(BackendCorrectnessTest, smallConv) {
   EXPECT_TRUE(out1.isEqual(out2));
 }
 
+TEST_P(CPUOnly, convDKKC8Test) {
```

Comment:
The test creates a Conv and assumes it will be optimized into a convDKKC8, but future things could change and that may no longer occur. Perhaps this should be called `groupConvTest` instead?